fission/function.go

```diff
@@ -134,6 +138,41 @@ func getContents(filePath string) []byte {
 	return code
 }
 
+func getPodLogs(c *cli.Context) bool {
+	fnName := c.String("name")
+	if len(fnName) == 0 {
+		fatal("Need --name argument.")
+	}
+
+	queryURL, err := url.Parse(c.GlobalString("server"))
+	checkErr(err, "Error parsing the base URL")
+	queryURL.Path = fmt.Sprintf("/proxy/logs/%s", fnName)
+
+	req, err := http.NewRequest("POST", queryURL.String(), nil)
+	checkErr(err, "Error creating logs request")
+
+	httpClient := http.Client{Timeout: 5 * time.Second}
+	resp, err := httpClient.Do(req)
+	checkErr(err, "Get Logs request failed")
+
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		fmt.Println("Error in getting logs from POD directly, will check logstore, StatusCode:", resp.StatusCode)
+		return false
+	}
+
+	body, err := ioutil.ReadAll(resp.Body)
+	var data interface{}
+	json.Unmarshal(body, &data)
+	ldata, err := base64.StdEncoding.DecodeString(data.(string))
```

Comment:
I'm confused about this. What does the body look like? Why use json.Unmarshal for just a string? Can you simply do `base64.StdEncoding.Decode(body)`?